server/server.js

```diff
@@ -0,0 +1,19 @@
+// initialize the express app
```

Comment:
this file has the same function as the `PlatePlannerServer.js` file so you should merge this file with `PlatePlannerServer.js`

Reply:
WIP.